tests/telemetry/test_writer.py

```diff
@@ -341,6 +340,7 @@ def test_update_dependencies_event_when_disabled(telemetry_writer, test_agent_se
             assert events[0]["request_type"] != "app-dependencies-loaded"
 
 
+@flaky(1735812000)
 def test_update_dependencies_event_not_stdlib(telemetry_writer, test_agent_session, mock_time):
```

Comment:
should we set `filter_heartbeat_events` to `False` for one of these tests to assert that the heartbeat should still be produced even when other events are? Maybe this is covered in another test and I'm missing it

Reply:
This test is straight up broken. Currently it asserts a heartbeat event is received (instead of checking for the [dependencies](https://github.com/DataDog/dd-trace-py/blame/9d5ff05ec61e9fdb9c331c90b0882e15a5c28ffb/tests/telemetry/test_writer.py#L355) event). 

Instead of setting `filter_heartbeat_events` to `False` I fixed the test case and marked it as "flaky" but it should honestly be marked as broken.

Reply:
`pytest.mark.skip` might be more appropriate here.

Reply:
I'm less concerned with this test in particular, more that we have at least one test that covers the regression